dowhy/timeseries/temporal_graph.csv

```diff
@@ -0,0 +1,6 @@
+node1,node2,time_lag
```

Comment:
in general, not a good idea to add datasets to a library. 
Can you move this dataset to the example_notebooks/datasets/timeseries folder.